### PR TITLE
tests: linktype_name test

### PR DESCRIPTION
**Pull request by jlucovsky**
Continuation of #1993  

Issue: 6954

Ensure that the linktype_name is included in the alerts.

Updates:
- Modify existing test cases to expand the range of linktype name values.
- Remove unnecessary changes to existing tests

## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6954

Suricata PR: https://github.com/OISF/suricata/pull/11564

### CODE REVIEW

File: tests/decode-chdlc-01/test.yaml [hunk 1]
```diff
@@ -5,32 +5,32 @@ requires:
 
 checks:
 
-    - filter:
```
Comment (catenacyber) [inline tests/decode-chdlc-01/test.yaml:8]: Why is this diff so big ?

File: tests/vxlan-decoder-02/test.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 requires:
   features:
     - HAVE_LIBJANSSON
+
```
Comment (catenacyber) [inline tests/vxlan-decoder-02/test.yaml:4]: nit

File: tests/decode-chdlc-02/suricata.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        - files:
+            force-magic: no
+        - flow
+        - stats
+app-layer:
+  protocols:
+    http:
+      enabled: yes
+      libhtp:
+         default-config:
```
Comment (catenacyber) [inline tests/decode-chdlc-02/suricata.yaml:23]: Do we need all this ?

File: tests/decode-chdlc-02/test.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+requires:
+
+  min-version: 8
+
+pcap: ../decode-chdlc-01/hdlc-http_1tx.pcap
```
Comment (catenacyber) [inline tests/decode-chdlc-02/test.yaml:5]: Why do we need a new test ? 

We can have `min-version: 8` on a single check

**Review by jufajardini — CHANGES_REQUESTED** [review]
Could you please add the ticket reference to the readme files, too?

**Comment by jlucovsky** [comment]
Continued in #2006

**Pull request closed without merging**